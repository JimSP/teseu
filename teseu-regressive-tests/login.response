status-code:200
response-headers:Date=[Tue, 28 Apr 2020 01:38:50 GMT],Content-Length=[721],Content-Type=[application/json],Access-Control-Allow-Headers=[Token,Origin,X-Requested-With,Content-Type,Accept,X-Access-Token],Access-Control-Allow-Methods=[GET, POST, PUT, DELETE, OPTIONS, HEAD],Access-Control-Allow-Credentials=[true],Access-Control-Allow-Origin=[*]
response-body:{"clienteAtivo":"A","emailValidado":"S","idCliente":413,"idTipoCliente":1,"nomeCliente":"lumena - Corretora","nomeTipoCliente":"Associado","status":200,"token":{"type":"string","value":"eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2Iiwia2lkIjpudWxsLCJjdHkiOiJKV1QifQ..YJJYiGhTushFCFE8LNgiEA.fqTgZb9tbgyitjZYy0JcLMVzj2c9Uf_5k2Duqiiya0cxlbGhz5zPTGeT7eUIwvHW-01htzwvhUWqENdqvcu43XUR2bznUJX4-UlMEldCoVUfjkkOFyJbwr_BMQBJ0uhS8cHvPFbcgzy_j9rfKQIwkuewbXyYi9lbisTRm4haacJi6OIPptPSwzl8Wh_LAkQxOTWYPl3WP0HzzU2rq3dAK3NqfnT8nRGh1JTECCecpPyf6NHf-aTfaq0kKzzgBvyc5irbxFimjsR90qAEa7ED5BINU8YghxnROmgiKRvjSEXIUy0SchVVOjBdrfPuPSNuHChY2ZaF-FS5-ItdHSAaeVlq4o-ypc9ZAIxqk0TApvFhnw8QkGzDvIBjL2A_Flk8cMz7npEFB9_K7ehvbqo1lw.FpXUiLCkgNl-9BmQk0QAOw"}}
