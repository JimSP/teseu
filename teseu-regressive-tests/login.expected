200=#{context.get("login.httpStatus")}